src/typeDefs/schema.graphql

```diff
@@ -157,6 +158,35 @@ input SubmissionSearchFilters {
   offset: Int
 }
 
+type AuditLogs {
+  id: ID!
+  actionType: ActionType!
+  tableName: TableName!
+  schemaVersion: Float!
+  jsonData: String!
```

Comment:
@ermish my research lead me to using `String!` for JSON data since there is no scalar type yet for json data (or we have to created it ourself?)

Reply:
I think that's right